package.json

```diff
@@ -1,27 +1,29 @@
 {
-    "name": "assertion-error"
```

Comment:
It feels so good to have this file indented like this 😆

Reply:
Yeah this is what happens automatically. I can't stand pre-line commas anyway! D: